pkg/scalers/scalersconfig/typed_config.go

```diff
@@ -279,7 +280,14 @@ func setConfigValueMap(params Params, valFromConfig string, field reflect.Value)
 	split := strings.Split(valFromConfig, elemSeparator)
 	for _, s := range split {
 		s := strings.TrimSpace(s)
-		kv := strings.Split(s, elemKeyValSeparator)
+		s = removeChars(s, []string{"{", "}", "\""})
```

Comment:
I would exclude `expressionAttributeNames` too for now and let it be parsed by the original imperative `json` parser. Parsing `json` is tricky.